bootstrap/kubeadm/main.go

```diff
@@ -53,9 +55,11 @@ func init() {
 
 	_ = clientgoscheme.AddToScheme(scheme)
 	_ = clusterv1alpha3.AddToScheme(scheme)
+	_ = clusterv1alpha4.AddToScheme(scheme)
 	_ = expv1alpha3.AddToScheme(scheme)
-	_ = kubeadmbootstrapv1alpha2.AddToScheme(scheme)
+	_ = expv1alpha4.AddToScheme(scheme)
 	_ = kubeadmbootstrapv1alpha3.AddToScheme(scheme)
```

Comment:
I'm not sure we need to add both 3 and 4 to the scheme. We can just do 4, right?